### PR TITLE
Pubmed fixes: Date & Target

### LINKED ISSUE #2288: Pubmed XML date parsing
Is broken when the month is presented as a number. See https://forums.zotero.org/discussion/86333/pubmed-translator-misinterpreting-publication-date#latest

### PR DESCRIPTION
**Pull request by adam3smith**
closes #2288
fixes 
https://forums.zotero.org/discussion/86662/unable-to-save-multiple-files-with-new-pubmed-and-zotero-connector#latest

### CODE REVIEW

File: PubMed XML.js [hunk 1]
```diff
@@ -180,7 +180,12 @@ function doImport() {
 					var year = ZU.xpathText(pubDate, 'Year');
 
 					if (day) {
-						newItem.date = month + " " + day + ", " + year;
+						if (month && /\d+/.test(month)) {
```
Comment (dstillman) [inline PubMed XML.js:183]: Could we just add a comment showing two examples of month values this is expecting?

**Review by dstillman — COMMENTED** [review]
It looks like [a PubMed XML test](https://travis-ci.com/github/zotero/translators/builds/208338647#L497) needs to be updated as well to reflect the new date format.

**Comment by adam3smith** [comment]
Gene Ontology: the website seems to be going very strong still, but I'm not really seeing where they cite/link to literature. I'll leave this alone for now, but might also be a candidate for deletion.

**Comment by dstillman** [comment]
Thanks!

**Pull request merged**